src/app/components/PageLayout/index.tsx

```diff
@@ -1,74 +1,47 @@
 import { FC, PropsWithChildren, ReactNode } from 'react'
 import { Header } from './Header'
 import { Footer } from './Footer'
-import Box from '@mui/material/Box'
 import { useScreenSize } from '../../hooks/useScreensize'
-import { styled, useTheme } from '@mui/material/styles'
 import { BuildBanner } from '../BuildBanner'
 import { useScopeParam } from '../../hooks/useScopeParam'
 import { NetworkOfflineBanner, RuntimeOfflineBanner, ConsensusOfflineBanner } from '../OfflineBanner'
 import { Search } from '../Search'
 import { useIsApiReachable } from '../OfflineBanner/hook'
+import { useTheme } from '@mui/material/styles'
 
 interface PageLayoutProps {
   mobileFooterAction?: ReactNode
 }
 
-export const StyledMain = styled('main')({
-  minHeight: '75vh',
-})
-
 export const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({ children, mobileFooterAction }) => {
-  const theme = useTheme()
   const { isMobile, isTablet } = useScreenSize()
   const scope = useScopeParam()
   const isApiReachable = useIsApiReachable(scope?.network ?? 'mainnet').reachable
+  const theme = useTheme()
 
   return (
     <>
       <BuildBanner />
       <NetworkOfflineBanner />
       {scope && scope.layer !== 'consensus' && <RuntimeOfflineBanner />}
       {scope && scope.layer === 'consensus' && <ConsensusOfflineBanner />}
-      <Box
-        sx={{
-          minHeight: '100vh',
-        }}
-      >
+      <div className="min-h-screen">
         <Header />
-        <Box
-          sx={
-            isMobile
-              ? {
-                  border:
-                    theme.palette.background.default !== theme.palette.layout.border
```

Comment:
note: we are not migrating this comparison. not sure why we omit border for mainnet only. probably some old (blue theme) search related thing? https://github.com/oasisprotocol/explorer/pull/524